R/setup.R

```diff
@@ -3,15 +3,14 @@ options(
   tidymodels.dark = TRUE, # Tidymodels CV output compatible with dark theme
   dplyr.summarise.inform = FALSE, # Disable messages when aggregating
   scipen = 99, # Disable scientific notation
-  knitr.kable.NA = "", # Replace NA with empty string in knitr::kable
-  java.parameters = "-Xmx10g" # Pre-allocate memory for Java JDBC driver
```

Comment:
This is no longer needed now that #70 is merged, as we no longer use the RJDBC driver.